lib/lwd/lwd.mli

```diff
@@ -67,6 +67,9 @@ val peek : 'a var -> 'a
 (** Observe the current value of the variable, without any dependency
     tracking. *)
 
+val map_and_set : ('a -> 'a) -> 'a var -> unit
```

Comment:
For inspiration, the stdlib has `Map.update`.

Reply:
That does read better.